src/validation.cpp

```diff
@@ -2803,11 +2746,7 @@ bool CChainState::ActivateBestChainStep(BlockValidationState& state, const CChai
     }
     m_mempool.check(&CoinsTip());
 
-    // Callbacks/notifications for a new best chain.
-    if (fInvalidFound)
-        CheckForkWarningConditionsOnNewFork(vpindexToConnect.back());
```

Comment:
Nit: Since we're touching this, perhaps a comment could be added to `vpindexToConnect` w/re how it's ordered for future clarity?

Reply:
Thanks. Added new commit with comment